README.md

```diff
@@ -91,12 +121,12 @@ and the GraphQL sponsors.
 
 This module can be summarized as follows:
 
-A Drupal 8 subsystem (REST) and a Symfony subsystem (Serialization) use a touchy
+A Drupal 8 subsystem (routing) and a Symfony subsystem (Serialization) use a touchy
 Drupal 8 subsystem (Typed Data) to describe the output of another Drupal 8
 subsystem (Entity).
 
 It gets worse--the use cases of this module are most applicable when said entity
-output comes by way of the REST and Serialization systems.
+output comes by way of the routing and Serialization systems.
```

Comment:
Overall I think switching this to talk about routing more makes sense, but since the practical use of the module is still defining serialized output probably created via REST module, we should bring back some mention. I will do that in follow-up.